### PR TITLE
Migrate DocumentIntelligence tests 

### PR DESCRIPTION
**Pull request by samvaity**
Passing Live Pipeline - https://dev.azure.com/azure-sdk/internal/_build/results?buildId=3943117&view=results

### CODE REVIEW

File: sdk/documentintelligence/tests.native.yml [hunk 1]
```diff
@@ -12,5 +13,7 @@ extends:
     CloudConfig:
       Public:
         SubscriptionConfiguration: $(sub-config-azure-cloud-test-resources)
+        ServiceConnection: azure-sdk-tests
+        SubscriptionConfigurationFilePath: eng/common/TestResources/sub-config/AzurePublicMsft.json
```
Comment (g2vinay) [inline sdk/documentintelligence/tests.native.yml:17]: Change this to:
          SubscriptionConfigurationFilePaths: 
            - eng/common/TestResources/sub-config/AzurePublicMsft.json

Comment (samvaity) [reply sdk/documentintelligence/tests.native.yml]: updated guidance ^^

**Review by g2vinay — APPROVED** [review]

**Review by mssfang — APPROVED** [review]

**Pull request merged**